docs/Configuration.md

```diff
@@ -938,3 +939,29 @@ Flags:
       --set stringArray             Set arbitrary component config property. The component has to be defined in the config file and the flag has a higher precedence. Array config properties are overridden and maps are joined, note that only a single (first) array property can be set e.g. -set=processors.attributes.actions.key=some_key. Example --set=processors.batch.timeout=2s (default [])
   -v, --version                     version for otelcol-sumo
 ```
+
+## Proxy Support
+
+Exporters that leverage the HTTP communication (currently all) respect the following proxy environment variables:
```

Comment:
I think we can reword it to focus on our exporter only

Reply:
Corrected, please check it ;)